src/envoy/mixer/README.md

```diff
@@ -66,7 +66,8 @@ This filter will intercept all HTTP requests and call Mixer. Here is its config:
          "mixer_server": "${MIXER_SERVER}",
          "mixer_attributes" : {
             "attribute_name1": "attribute_value1",
-            "attribute_name2": "attribute_value2"
+            "attribute_name2": "attribute_value2",
+            "quota.name": "RequestCount",
```

Comment:
No, they will be converted to protobuf field in mixer client. Please checkout this PR
https://github.com/istio/mixerclient/pull/44